toolchain/check/testdata/interop/cpp/no_prelude/multiple_imports.carbon

```diff
@@ -26,7 +26,7 @@ import Cpp library "file2.h";
 // CHECK:STDOUT: --- multiple_imports.carbon
 // CHECK:STDOUT:
 // CHECK:STDOUT: imports {
-// CHECK:STDOUT:   %Cpp: <namespace> = namespace file.%cpp_import.loc4, [template] {}
+// CHECK:STDOUT:   %Cpp: <namespace> = namespace file.%import_cpp.loc4, [template] {}
```

Comment:
The location is interesting, considering they are all being merged as you said. I'm not sure what that will do for diagnostics, if it will be hard to know which import is bringing in a clang error? Or maybe that is clear in diagnostics but we just don't have them all listed in semir? Then that might make tooling difficult though.

I think this is okay for now, but maybe we can think about improvements if this is a pain. Perhaps we could have multiple %Cpp imports and they'd get `.xyz` suffixes in semir.

Reply:
This is how all imported namespaces behave -- C++ isn't any different here. i.e., you should expect the same result from:

```
import Foo library "bar";
import Foo library "baz";
```

The name `Foo` can have only one location associated with it.

When it comes to _specific_ names, we can have more specific locations. We probably won't have that short-term for C++ due to complexities involves, and how `#include` semantics need to take priority. But a single name -- whether `Cpp`, `Foo`, or something else -- has a single location in SemIR.